From 3ef8b23b32b53cb679e755e2e863512e6228c841 Mon Sep 17 00:00:00 2001
From: Robin Gong <b38343@freescale.com>
Date: Wed, 30 Sep 2015 16:07:06 +0800
Subject: [PATCH 0831/1691] MLK-11660: spi: spi-imx: fix building error

commit 3ef8b23b32b53cb679e755e2e863512e6228c841 from
git://git.freescale.com/imx/linux-2.6-imx.git imx_4.1.15_1.0.0_ga

fix building error by d5150a37c14:

drivers/spi/spi-imx.c: In function 'spi_imx_can_dma':
drivers/spi/spi-imx.c:211:3: error: expected ')' before 'return'
   return true;
      ^
      drivers/spi/spi-imx.c:213:1: error: expected expression before '}' token
      4.1build error

Signed-off-by: Robin Gong <b38343@freescale.com>
---
 drivers/spi/spi-imx.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/spi/spi-imx.c b/drivers/spi/spi-imx.c
index e9e5f63..936065d 100644
--- a/drivers/spi/spi-imx.c
+++ b/drivers/spi/spi-imx.c
@@ -207,7 +207,7 @@ static bool spi_imx_can_dma(struct spi_master *master, struct spi_device *spi,
 	struct spi_imx_data *spi_imx = spi_master_get_devdata(master);
 
 	if (spi_imx->dma_is_inited &&
-		(transfer->len > spi_imx_get_fifosize(spi_imx))
+		(transfer->len > spi_imx_get_fifosize(spi_imx)))
 		return true;
 	return false;
 }
-- 
1.9.1

